tests/integration/docker/nodejs/nodejs22.x/serverless.yml

```diff
@@ -0,0 +1,30 @@
+service: docker-nodejs22-x-test
+
+configValidationMode: warn
```

Comment:
@DorianMazur I've had to set this to `warn` instead of `error` for the test to pass as serverless v3 does not have `nodejs22.x` as a supported runtime, hope this is fine.